lib/util/http/auth.ts

```diff
@@ -6,7 +6,20 @@ import {
 } from '../../constants';
 import type { GotOptions } from './types';
 
-export function applyAuthorization(inOptions: GotOptions): GotOptions {
+type AuthGotOptions = Pick<
```

Comment:
```suggestion
export type AuthGotOptions = Pick<
```
we should export all public types and all types passed to public functions

Reply:
I don't agree in this case, since we gonna use another type that must extend this one and we never gonna use the type itself. But okay.